ios/RNSScreenStackHeaderConfig.mm

```diff
@@ -572,6 +591,39 @@ + (void)updateViewController:(UIViewController *)vc
   navitem.leftBarButtonItem = nil;
   navitem.rightBarButtonItem = nil;
   navitem.titleView = nil;
+#ifdef RN_FABRIC_ENABLED
+  for (RNSScreenStackHeaderSubviewComponentView *subview in config.reactSubviews) {
+    switch (subview.type) {
+      case facebook::react::RNSScreenStackHeaderSubviewType::Left: {
+        //#if !TARGET_OS_TV
```

Comment:
Why is it commented?

Reply:
`@property (nonatomic) BOOL backButtonInCustomView;` is not implemented yet on Fabric.